client/src/components/ContentNavigator/ContentModel.ts

```diff
@@ -362,20 +336,21 @@ export class ContentModel {
   private getFileInfo(resourceId: string): {
     etag: string;
     lastModified: string;
+    contentType: string;
   } {
-    if (resourceId in this.fileTokenMaps) {
-      return this.fileTokenMaps[resourceId];
+    if (resourceId in this.fileMetadataMap) {
+      return this.fileMetadataMap[resourceId];
     }
     const now = new Date();
     const timestamp = now.toUTCString();
-    return { etag: "", lastModified: timestamp };
+    return { etag: "", lastModified: timestamp, contentType: "text/plain" };
```

Comment:
nit: DEFAULT_FILE_CONTENT_TYPE

Reply:
Done in https://github.com/sassoftware/vscode-sas-extension/pull/980/commits/1ee77cc65681082fc3b202e55e265821d23a11db